rom/dev/src/fht.rs

```diff
@@ -176,3 +178,13 @@ pub fn store(fht: FirmwareHandoffTable) {
     caliptra_common::print_fht(&fht);
     slice.copy_from_slice(fht.as_bytes());
 }
+
+pub fn load_from_dccm() -> CaliptraResult<FirmwareHandoffTable> {
+    let slice = unsafe {
```

Comment:
cant we use the function try_load in common?

Reply:
Done.